### ISSUE TITLE
[SUGGESTION] | "Abdicate to join the Kirin Tor"

### ISSUE THREAD

**Issue by killshrooms** [Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth#603]
----------------------------------------------------------------------------------------------------------
Higher tier mages (at least T6/T7?) should have the option to give their throne up to the appropriate heir to join as a member of the Kirin Tor, effectively making them a functioning part of the magocracy. If they happen to be elected as leader of the Kirin Tor, all living members of their dynasty get a powerful monthly prestige gain for as long as they remain in power.

**Comment by DukeJeans** [comment]
personal note: arcane pope

**Comment by mdbpr-iscteiulpt** [comment]
Yeah i was asking myslef why not just add a bishporic mechanic to dalaran. you could even recomend realm mages to the council. Remove the election funds or make them not matter and we have a pretty fun dalaran council.
